### PR TITLE
Chore: Issue 1647 - Dependency updates for multiple packages

### PR DESCRIPTION
**Pull request by bhushan-ebi**

### CODE REVIEW

File: tools/vf-component-library/src/site/_includes/layouts/components.njk [hunk 1]
```diff
@@ -283,7 +283,7 @@ This approach is best for bare-bones Nunjucks environments, such as precompiled
 %}{% endraw %}
                 </code></pre>
               </details>
-              {% set angularComponents = "vf-badge, vf-banner, vf-hero, vf-button, vf-blockquote" %}
+              {% set angularComponents = "vf-badge, vf-banner, vf-hero, vf-button, vf-blockquote, vf-tabs" %}
```
Comment (sandykadam) [inline tools/vf-component-library/src/site/_includes/layouts/components.njk]: should we have this change in Tabs angular PR?

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -12,9 +12,9 @@ jobs:
         node-version: [14.x, 16.x]
```
Comment (sandykadam) [inline .github/workflows/nodejs.yml]: I think you should update this node version to 16 & 18. Then I think builds will work for both PRs

File: tools/vf-component-library/src/site/_includes/layouts/components.njk (hunk 1 above)
Comment (bhushan-ebi) [reply tools/vf-component-library/src/site/_includes/layouts/components.njk]: Added this change to the Tabs PR.

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (bhushan-ebi) [reply .github/workflows/nodejs.yml]: This change is done.

**Comment by bhushan-ebi** [comment]
Hi @sandykadam , If I use node-version: [18.x] Node CI build succeeds but other checks (Header, Pages changed, Redirect rules, netlify) fail. But when I use node-version: [16.x, 18.x] Node CI build as well as all these checks fail. Any suggestions?

**Pull request closed without merging**